main/menu.js

```diff
@@ -1,35 +1,12 @@
 'use strict';
```

Comment:
This file is also used on macOS to create macOS specific menu, this menu is not visible under Windows.